ignition/model/lifecycle.py

```diff
@@ -2,16 +2,19 @@
 STATUS_COMPLETE = 'COMPLETE'
 STATUS_FAILED = 'FAILED'
 STATUS_UNKNOWN = 'UNKNOWN'
+LIFECYCLE_MESSAGE_VERSION = "1.0.0"
```

Comment:
The version number seems to be repeated in numerous places i.e. for each message send capability. Is it the intention that the version numbers should be independently changeable for each message send capability i.e. request queue vs job queue, etc?

Reply:
Yes, independent message versions for each capability. There is no central file where we could have put a common message-version string for each capability, so I put it in each file with it's own version name.

Reply:
ok